keyboards/j80/config.h

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2019 JER
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+#define VENDOR_ID 0x20A0
+#define PRODUCT_ID 0x422D
+#define DEVICE_VER 0x0001
+#define MANUFACTURER JER
+#define PRODUCT J80
+
+#define MATRIX_ROWS 7
+#define MATRIX_COLS 14
+
+//                         0   1   2   3   4   5   6   7   8   9   A   B   C   D
+#define MATRIX_ROW_PINS { B1, B2, B3, B5, B6, B7, B0 }
```

Comment:
I usually prefer that row 0 and col 0 are b0, a0. This makes it so that K00 is always the reset key, but it's not a requirement. Also, I have encountered a small handful of boards where the reset key is not a0, b0.

Reply:
The matrix on this PCB is kinda wonky. A0 is only used on two keys and B0 is only used on the top right cluster. Started with B1 and A1 as those are the first rows and columns when starting from Escape. Can change this around if necessary, but it will be a lot of work.
![matrix](https://user-images.githubusercontent.com/46304257/69739176-888c7000-1137-11ea-8b8e-fda8433fd54b.png)

Reply:
> This makes it so that K00 is always the reset key

Yeah, but you can change that with a define. :)